### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.12 esercizio 1 (25 novembre)

### DISCUSSION THREAD

**Discussion by alem1105** [sapienzastudentsnetwork/calcolo-delle-probabilita#333] [Esercizi M-Z]
<img width="1095" alt="image" src="https://github.com/user-attachments/assets/06252e24-af55-4a6d-8082-eb1721d8dd32">

**Comment by AglaiaNorza** [comment]
![es](https://github.com/user-attachments/assets/945b20e7-8c39-4b4c-883c-2b0190cd68c6)
